README.md

```diff
@@ -4,7 +4,7 @@
 [![Starve.io Extensions Reddit](http://i.imgur.com/ccMEPJK.png "Starve.io Extensions Reddit")](https://www.reddit.com/r/starveio/comments/6xiaec/starveio_extensions_github_compliments_to/)
 
 # TO USE / INSTALL
-1. Install [Tampermonkey](https://tampermonkey.net)
+1. Install [Tampermonkey](https://chrome.google.com/webstore/detail/tampermonkey/dhdgffkkebhmkfjojejmpbldmpobfkfo)
```

Comment:
Tampermonkey is compatible with Mozilla Firefox, Opera Next, Dolphin Browser, UC Browser, Google Chrome, Chromium, etc.